src/unitxt/inference.py

```diff
@@ -425,6 +423,24 @@ class WMLInferenceEngine(
     data_classification_policy = ["public", "proprietary"]
     parameters: Optional[WMLInferenceEngineParams] = None
 
+    _client: Any = None
```

Comment:
It's not but I just wanted to put it here so that all class' unique attributes are in one place. I marked it as a protected attribute so that users do not pass it on their own

Reply:
I see so just use InternalField which will de facto will protect it

Reply:
```
from .dataclass import InternalField

...
_client: Any = InternalField(default=None)

```

Reply:
Thanks, I forgot that we have something like this